src/core/iomux/io_mux_call.cpp

```diff
@@ -416,6 +416,10 @@ void io_mux_call::blocking_loops()
             __log_func("going to sleep (elapsed time: %d sec, %d usec)", m_elapsed.tv_sec,
                        m_elapsed.tv_usec);
 
+            if (check_all_offloaded_sockets()) {
```

Comment:
Not needed. The manual triggering mechanism inside DOCA solves the race.

Reply:
but i use a patch in doce to disable manual triggering as well

Reply:
lets discuss this one

Reply:
Removed